crates/compression/src/ports.rs

```diff
@@ -56,10 +82,22 @@ pub struct MessageInfo {
     pub data: Vec<u8>,
 }
 
-/// Temporal registry evictor state storage,
-/// currently backed by a `DaCompressionTemporalRegistryEvictor*`
-/// columns in the offchain database.
```

Comment:
Not sure why this is removed? It's quite helpful in finding the backing storage. I understand that it breaks the illusion of not depending on the other crate, but in reality they are tightly coupled.

Reply:
The trait should be defined without implementation details. The name of the table that is used under the hood inside of the `fuel-core` an be different in the future, or implementation can be different=) We don't need this information here.

We need to describe how this trait could be used, but not how it is used.